src/js/modules/core/components/PageHeader.js

```diff
@@ -51,21 +45,16 @@ const styles = {
   },
 };
 
-const PageHeader = ({classes, sectionsWithSubsections}) => {
+const PageHeader = ({ classes, location, sectionsWithSubsections }) => {
   return (
     <div className={classes.PageHeader}>
-      <div className={classes.theSpectatorLogo}>The Spectator</div>
-      <ul className={classes.sectionLinksNav}>
-        {sectionsWithSubsections.map((topLevelSection) => {
-          return (
-            <li key={topLevelSection.id} className={classes.sectionListElement}>
-              <Link to={`/${topLevelSection.slug}`} className={classes.sectionLink}>
-                {topLevelSection.name}
-              </Link>
-            </li>
-          )
-        })}
-      </ul>
+      {
+        location.pathname === '/' ? (
```

Comment:
PageHeader isn't rendered in Routes, it's rendered outside of <Switch>, so I just check the location.